src/librustc_interface/Cargo.toml

```diff
@@ -34,6 +34,7 @@ rustc_errors = { path = "../librustc_errors" }
 rustc_plugin = { path = "../librustc_plugin", package = "rustc_plugin_impl" }
 rustc_privacy = { path = "../librustc_privacy" }
 rustc_resolve = { path = "../librustc_resolve" }
+rustc_error_codes = { path = "../librustc_error_codes" }
```

Comment:
Now that you've done this, it might be the case that references to other `rustc_` crates above are redundant; try removing some and see if that's the case?

Reply:
Ah good point. Let's see!

Reply:
So apparently not. They're all used independently of the error codes...